skeleton/app.js

```diff
@@ -17,6 +17,14 @@ import Squirrel from './squirrel'; // DEPRECATED
 const { app, BrowserWindow, dialog } = electron;
 const { join } = path;
 
+if (parseInt(process.versions.electron.split('.')[0]) >= 5) {
```

Comment:
Can you check if your code is still needed? I think code from App.contructor (line 31-37) should resolve your problem.